junit-jupiter-api/src/main/java/org/junit/jupiter/api/parallel/Execution.java

```diff
@@ -99,4 +99,10 @@
 	 */
 	ExecutionMode value();
 
+	/**
+	 * The reason of using a non-default execution mode.
```

Comment:
```suggestion
	 * The reason for using the selected execution mode.
```

Please also add appropriate `@since` and `@API` declarations.